### ISSUE TITLE
 Thermostat card looks like a sad smiley if entity is off

### ISSUE THREAD

**Issue by casperklein** [home-assistant/frontend#6025]
## Checklist

- [x] I have updated to the latest available Home Assistant version.
- [x] I have cleared the cache of my browser.
- [x] I have tried a different browser to see if it is related to my browser.

## The problem

After Upgrading from **0.109.6** to **0.110.2** the thermostat looks "sad", when turned off.

Version: 0.109.6

![109](https://user-images.githubusercontent.com/590174/82760609-e652f980-9df4-11ea-8dce-141584e58453.PNG)

Version 0.110.2

![110 2](https://user-images.githubusercontent.com/590174/82760622-01256e00-9df5-11ea-9447-90a20a46e55b.PNG)

## Expected behavior

The thermostat card should look like before in version 0.109.6.

## Steps to reproduce

Upgrade from 0.109.6 to 0.110.2.

## Environment

- Home Assistant release with the issue: 0.110.2
- Last working Home Assistant release (if known): < 0.110.0
- Browser and browser version: Firefox 76.0.1 / Chrome 81.0.4044.138
- Operating system: Windows 10 / Android 10

## State of relevant entities

```yaml
 climate.wohnzimmer state off

Attributes:
hvac_modes: heat, off
min_temp: 8
max_temp: 28
preset_modes: eco, comfort
current_temperature: 18
temperature: 0
preset_mode: null
battery_low: false
device_locked: false
locked: false
battery_level: 70
holiday_mode: false
summer_mode: false
window_open: false
friendly_name: Wohnzimmer
supported_features: 17
```

## Problem-relevant configuration

```yaml

```

## Javascript errors shown in your browser console/inspector

```txt

```

## Additional information

- Deleting and re-adding the thermostat card does not fix the issue
- Even before adding the thermostat card, the preview is already wrong + there are the letters "ff" on the left:
![110 2-preview](https://user-images.githubusercontent.com/590174/82760787-4c8c4c00-9df6-11ea-81a9-d474c79f6ab0.PNG)

I found out, that there seems to be additional HTML code. The blue marked line is not present in in 0.109.6. If I remove that line with Firefox developer tools, it looks fine.

![110 2-html](https://user-images.githubusercontent.com/590174/82760978-87db4a80-9df7-11ea-99aa-0508ae526388.PNG)

**Comment by bramkragten** [comment]
@thomasloven any ideas?

**Comment by thomasloven** [comment]
Probably happens because the target temperature (0) is outside of the range (8-28).

**Comment by thomasloven** [comment]
I thought i had fixed that in the slider...

**Comment by casperklein** [comment]
Thanks for looking into that. If there is something can test, please let me know.

**Comment by casperklein** [comment]
Fixed in 0.110.3. Thank you!
